### PR TITLE
#184651324 App Logout Functionality

### PR DESCRIPTION
**Pull request by bahati10**
### **What does this PR do?**
Adds log-out functionality to the project
##### **Description of Task to be completed?**
Users should be able to log out of the application
As a user, I want to click on the "logout" link So 
that I get logged out of the application When a 
user clicks on the logout link, they should be 
signed out of the application with their session ended.

#### How should this be manually tested?
- First Clone the repository  
- Navigate to the branch project 
- use **`npm install`** to install the required dependencies.
- to test use **`npm run test`** and see the coverage.
- use **`npm start`** to start the server
- Navigate to the project URL `http://localhost:3000`
- Login with email and you will be redirected to 
  the protected page
- Click the **Logout** button to log out.

#### What are the relevant pivotal tracker/Trello stories?
[#184651324](https://www.pivotaltracker.com/n/projects/2633635)

### CODE REVIEW

**Review by ruberwa — DISMISSED** [review]

**Review by heshima18 — DISMISSED** [review]

**Review by bstwagir — DISMISSED** [review]
Looks good.

File: README.md [hunk 1]
```diff
@@ -1,96 +1,11 @@
-# Team-Furebo-E-commerce-bn
+# Team-furebo-E-commerce-bn
 
-[![Maintainability](https://api.codeclimate.com/v1/badges/7322a671c14645284094/maintainability)](https://codeclimate.com/github/atlp-rwanda/Team-furebo-E-commerce-bn/maintainability) [![Test Coverage](https://api.codeclimate.com/v1/badges/7322a671c14645284094/test_coverage)](https://codeclimate.com/github/atlp-rwanda/Team-furebo-E-commerce-bn/test_coverage) [![Coverage Status](https://coveralls.io/repos/github/atlp-rwanda/Team-furebo-E-commerce-bn/badge.svg?branch=develop)](https://coveralls.io/github/atlp-rwanda/Team-furebo-E-commerce-bn?branch=develop)
```
Comment (furebo) [inline README.md]: Why have you deleted this ?

File: README.md [hunk 2]
```diff
@@ -1,96 +1,11 @@
...
-### Pre-Requisites <a name="pre-requisites"></a>
-
-* Node.js version 18.x
-* Docker
-* Postgres Database
-* ``` others will be mentioned later ```
-
-### File Structure <a name="structure"> </a>
-
-- `.circleci` This folder contains the configuration file for the integration of all the testing webhooks that verify any pushed code to develop. I.e. CircleCI, Coveralls, CodeClimate.
```
Comment (furebo) [inline README.md]: why have you deleted these project description in this file ?

File: package.json [hunk 3]
```diff
@@ -4,8 +4,8 @@
   "description": "[![atlp-rwanda](https://circleci.com/gh/atlp-rwanda/Team-furebo-E-commerce-bn.svg?style=svg)](https://app.circleci.com/pipelines/github/atlp-rwanda)",
   "main": "index.js",
   "scripts": {
-    "test": "nyc --reporter=lcov --reporter=text mocha src/tests/*.test.js --timeout 60000 --require @babel/register --exit && codeclimate-test-reporter < coverage/lcov.info",
-    "coverage": "nyc report --reporter=text-lcov | tee coverage.lcov | coveralls && ./node_modules/.bin/codeclimate-test-reporter < coverage/lcov.info",
```
Comment (furebo) [inline package.json]: Why have you removed these lines for running codeclimate when the tests run and the coverage on code climate ?

**Review by furebo — DISMISSED** [review]
Please rebase from develop and make sure you don't delete the completed works.

**Review by nshutip — DISMISSED** [review]

**Review by ruberwa — DISMISSED** [review]

**Review by JulesNIYOMWUNGERI — DISMISSED** [review]

**Review by Raphaela664 — APPROVED** [review]
Great job on this pull request

**Review by nshutip — APPROVED** [review]
Remember to ignore the coverage folder in your next PRs.

**Pull request merged**